app/src/androidTest/java/com/mutualmobile/barricade/sample/BarricadeActivityTest.java

```diff
@@ -0,0 +1,80 @@
+package com.mutualmobile.barricade.sample;
+
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.contrib.RecyclerViewActions;
+import android.support.test.filters.LargeTest;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.util.Log;
+import com.mutualmobile.barricade.Barricade;
+import com.mutualmobile.barricade.BarricadeConfig;
+import com.mutualmobile.barricade.activity.BarricadeActivity;
+import com.mutualmobile.barricade.response.BarricadeResponse;
+import com.mutualmobile.barricade.response.BarricadeResponseSet;
+import com.mutualmobile.barricade.utils.TestAssetFileManager;
+import java.util.HashMap;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+@RunWith(AndroidJUnit4.class) @LargeTest public class BarricadeActivityTest {
+  @Rule public ActivityTestRule<BarricadeActivity> mActivityRule =
+      new ActivityTestRule<>(BarricadeActivity.class);
+
+  private static Barricade barricade;
+
+  @BeforeClass public static void setup() {
+    barricade =
+        new Barricade.Builder(BarricadeConfig.getInstance(), new TestAssetFileManager()).install();
```

Comment:
`TestAssetFileManager` is for tests that run on the JVM. Since these tests will run on Android, use `AndroidAssetFileManager` instead.